shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/event/MetaStoreEvent.java

```diff
@@ -185,6 +211,10 @@ public String getQualifiedTableName() {
     return String.join(".", getDatabaseName(), getTableName());
   }
 
+  public String getQualifiedReplicaTableName() {
+    return String.join(".", getReplicaDatabaseName(), getReplicaTableName());
```

Comment:
Ah, here it is again, perhaps it does justify a higher level `TableDatabaseNameJoiner` util class?

Reply:
Ok. I have created that joiner.